albumentations/augmentations/transforms.py

```diff
@@ -1714,27 +1714,37 @@ def __init__(
         self,
         scale_min=0.25,
         scale_max=0.25,
-        interpolation=cv2.INTER_NEAREST,
+        interpolation=None,
         always_apply=False,
         p=0.5,
     ):
         super(Downscale, self).__init__(always_apply, p)
+        if interpolation is None:
+            self.interpolation = cv2.INTER_NEAREST, cv2.INTER_NEAREST
+            warnings.warn(
```

Comment:
I think it might not be obvious to a user how to correctly set interpolation modes both for downscaling and upscaling using a tuple. Maybe use some class to make this setting more self-explainable, e.g. 

```
Downscale(
    ...,
    interpolation=Interpolation(downscale=cv2.INTER_NEAREST, upscale=cv2.INTER_NEAREST)
)
```
?

Reply:
@creafz check this again